kubelet/metadata.csv

```diff
@@ -1,4 +1,9 @@
 metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+kubernetes.cpu.load.10s.avg,gauge,,,,Container cpu load average over the last 10 seconds,0,kubelet,k8s.cpu.load.10s
+kubernetes.cpu.system.total,gauge,,nanocore,,The number of cores used for system time,0,kubelet,k8s.cpu.system
+kubernetes.cpu.user.total,gauge,,nanocore,,The number of cores used for user time,0,kubelet,k8s.cpu.user
+kubernetes.cpu.cfs.throttled.periods,gauge,,,,Number of throttled period intervals,-1,kubelet,k8s.cpu.throttled.periods
+kubernetes.cpu.cfs.throttled.seconds,gauge,,,,Total time duration the container has been throttled,-1,kubelet,k8s.cpu.throttled.sec
```

Comment:
https://github.com/DataDog/integrations-core/pull/2595#discussion_r234214305 is still true, we should set the unit to `seconds`

Reply:
this should be `second` singular :)

Reply:
Wait, we test metadata? That's pretty cool! 😎